godotproject/spike.gd

```diff
@@ -0,0 +1,6 @@
+extends Area2D
+
+
+func _on_Spike_body_entered(body):
```

Comment:
Do we want the player and all other bodes with death method to immediately die to spike damage? In my head I was going to implement a knockback for when players ran into enemies and intended to do the same for spikes. What are your thoughts?

Reply:
I agree, they should take damage and not die immediately. I didnt think your PR for damage was released yet. rn the ant can one-shot me

Reply:
The damage is released, I just set the ant to one shot since I hadn't implemented the visual invulnerable / take damage portion. The players health is set to 3 and the ant attack does 3 damage :)

Reply:
Oh I see. Yeah will do, and then we can decrement the amount of damage when the HUD and invuln stuff is added

Reply:
Yep, will make things easier when we add the visuals in. Thanks